docs/source/model_design/vivarium_model_components/GBD_disease_health/index.rst

```diff
@@ -130,20 +109,58 @@ say that the health state and therefore the disability weight is the same for bo
 Impairments
 +++++++++++
 
-In some cases, a medical condition or sequela might have multiple causes. For example, heart failure. 
-It is caused by both IHD and HHD (and others!) and has sequelae associated with both causes. 
-We even learned above that it has the exact same health state in both causes. Heart failure 
-is an example of an impairment. 
+Sometimes, GBD has a lot of information about a health state - such as severe heart failure in the 
+example above - that the modelers want to utilize. However, in the process of normal GBD 
+modeling, this wouldn't be possible. Usually, the cause level information is calculated 
```

Comment:
```suggestion
modeling, this wouldn't be possible. Usually, the cause is estimated
```

This feels a little less vague

Reply:
The phrase "the cause is estimated" doesn't exactly make sense to me, because how do you quantify a cause? Do you mean the cause prevalence is estimated? Or DALYs? Or what?

Reply:
Fair question, and I'm not exactly sure. My sense is that incidence, prevalence and remission are estimated at the cause level, then _prevalence_ specifically is divided into the sequelae, and finally YLDs are calculated at the sequela level. Would be good to confirm this (that there are no incidence or remission estimates for sequelae).